kytos/core/auth.py

```diff
@@ -87,6 +95,76 @@ def _generate_token(cls, username, time_exp):
             algorithm=cls.encode_algorithm,
         )
 
+    def create_user(self):
+        """Create user to database"""
+        req = request.json
+        utc_now = datetime.datetime.utcnow()
+        data = {
+            "username": req["username"],
+            "email": req["email"],
+            "password": req["password"],
+            "inserted_at": utc_now,
+            "updated_at": utc_now,
+        }
+        try:
+            user = UserDoc(**data)
+        except ValueError as error:
+            raise error
+        result = self.db.users.insert_one(user.dict())
+        return result
+
+    def delete_user(self, uid):
+        """Delete user from database"""
+        result = self.db.users.delete_one({"id": uid})
```

Comment:
For deletions, we need soft deletions instead of hard deletions for `users` for now. When soft deleting a user you need to update `deleted_at` and `state='inactive'`. An `inactive` user shouldn't be able to login on `auth/login/`